reference/public-cloud/azure.md

```diff
@@ -6,7 +6,7 @@ canonical_url: '/reference/public-cloud/azure'
 
 ## About {{site.prodname}} on Azure
 
-While Azure does not support {{site.prodname}} networking, you can use
+While Azure does not support {{site.prodname}} networking without encapsulation, you can use
```

Comment:
I think this is also not true - VXLAN CrossSubnet works within a single CIDR.

I think we should just remove the "does not suport {{site.prodname networking" line altogether. @lwr20

Reply:
Updated.